components/datasets/metadata/form/published-layer/list/PublishedLayersList.js

```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+
+// Components
+import PublishedLayerCard from 'components/datasets/metadata/form/published-layer/card/PublishedLayerCard';
+
+// styles
+import './styles.scss';
+
+class PublishedLayersList extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = { layers: props.layers };
```

Comment:
Move `state` outside and you can get ride of the `constructor`

Reply:
@andresgnlez But how could I initialize it with the props values in that case... 🤔

Reply:
`state = { layers: this.props.layers }`